### DISCUSSION TITLE
Ender 3v2 4.2.2 Problem with the printing speed

### DISCUSSION THREAD

**Discussion by MIB205** [mriscoc/Ender3V2S1#1241] [General]
Hallo together. I installed The professional firmware yesterday on my ender 3v2 4.2.2. 
Leveling and adjusting worked very well. When I wanted to print, the Brim was printed with the right speed. As IT starts to prind the Objekt. IT startet to print very fast on the first Player. I had to pull IT Down to 20 % of the speed and IT Was still to fast.
The Brim in the next layer it continue in a slow speed. Later IT get again very fast.
Hast anyone an idea where i can Chance some settings oder install new files?
I will be thankfull for any Information.
Greats Michael

**Comment by mriscoc** [comment]
Hi, check your slicer settings and start G-code script. This firmware unlocks higher printing speed than the stock firmware, so if the slicer was configured with high speeds, maybe the stock firmware was limiting that.
